reinforcement_learning/Lesson-3-challenge-draft.ipynb

```diff
@@ -0,0 +1,342 @@
+{
```

Comment:
<p>Add a ";)" at the end of the bullet to play a little on the coincidence of Atari Breakout and Breakout Mentors</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/BreakoutMentors/Data-Science-and-Machine-Learning/pull/61/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>